Formula/wownero.rb

```diff
@@ -0,0 +1,59 @@
+class Wownero < Formula
+  desc "Official wallet and node software for the Wownero cryptocurrency"
+  homepage "https://wownero.org"
+  url "https://git.wownero.com/wownero/wownero.git",
+    tag:      "v0.8.0.2",
+    revision: "7d1693b1601cb4b3d56191953f403c92a1c51f64"
+  license "BSD-3-Clause"
+
+  bottle :unneeded
```

Comment:
Since it compiles it shouldn't be unneeded.

Reply:
What should I put there? Documentation leads me to believe that this is adjusted after being merged and binaries are built: https://docs.brew.sh/Bottles#creation

> Bottles are created using the Brew Test Bot, usually when people submit pull requests to Homebrew. The bottle do block is updated by maintainers when they merge a pull request.

Reply:
Yes